ajax/libs/datatables.net-fixedheader-dt/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "datatables.net-fixedheader-dt",
+  "version": "3.1.2",
+  "description": "FixedHeader for DataTables ",
+  "filename": "css/fixedHeader.dataTables.min.css",
+  "keywords": [
+    "fixed headers",
+    "sticky",
+    "DataTables",
+    "jQuery",
+    "table"
+  ],
+  "homepage": "https://datatables.net/extensions/fixedheader",
```

Comment:
Would README.md in git repo be better? I can't find info about this styling library in the current link.